javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,54 @@
+boughtCandyPrices = [];
```

Comment:
```suggestion
const boughtCandyPrices = [];
```
It's better to use const to declare these variables since they don't change